Sources/Views/MessagesCollectionView.swift

```diff
@@ -85,19 +85,28 @@ open class MessagesCollectionView: UICollectionView {
     
     private func setupGestureRecognizers() {
         let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleTapGesture(_:)))
-        tapGesture.delaysTouchesBegan = true
```

Comment:
If there is a need to keep this please let me know, but failing on the existing `UICollectionView` recognizers should be enough. This avoids a delay in dragging the `UISlider`.

Reply:
This could be an issue removing it

Reply:
Please feel free to do testing with and without it, I found that it created issues with the slider. Why was it added in the first place?